src/lib/pages/public-project/components/SocialMedia.tsx

```diff
@@ -1,27 +1,20 @@
 import { Flex } from "@chakra-ui/react";
-import {
-  FaTwitter,
-  FaGithub,
-  FaTelegram,
-  FaDiscord,
-  FaInfo,
-} from "react-icons/fa";
-import { MdLanguage } from "react-icons/md";
 
-import { IconButton } from "lib/components/button";
+import { CustomIconButton } from "lib/components/button/CustomIconButton";
+import type { ICONS } from "lib/components/icon/CustomIcon";
 import { AmpTrackSocial, AmpTrackWebsite } from "lib/services/amplitude";
 import type { Option, PublicDetail } from "lib/types";
 
-export const renderSocial = (name: string) => {
+export const renderSocial = (name: string): keyof typeof ICONS => {
```

Comment:
This is the identical issue to which I responded previously. Please implement the solution I provided.